### DISCUSSION TITLE
[Desktop] Failed to load url: ERR_ABORTED when inspecting state.loadingState

### DISCUSSION THREAD

**Discussion by KevinnZou** [KevinnZou/compose-webview-multiplatform#58] [General]
@DatL4g I discovered a bug while working on the construction of JsBridge. Specifically, I'm trying to listen to the loading state of a WebView by using LaunchedEffect:
```
// commonMain/web/WebView.kt
@Composable
fun WebView(
    state: WebViewState,
    modifier: Modifier = Modifier,
    captureBackPresses: Boolean = true,
    navigator: WebViewNavigator = rememberWebViewNavigator(),
    onCreated: () -> Unit = {},
    onDispose: () -> Unit = {},
) {
    val webView = state.webView

     //This code causes the loading process to fail.
    LaunchedEffect(state.loadingState) {
        Logger.d { "WebView loadingState: ${state.loadingState}" }
    }

    ActualWebView(
        state = state,
        modifier = modifier,
        captureBackPresses = captureBackPresses,
        navigator = navigator,
        onCreated = onCreated,
        onDispose = onDispose,
    )
}
```

However, I'm experiencing an issue where the Desktop version fails to load initially when LaunchedEffect is used. It only succeeds after reloading. This problem can be reproduced in any branch and it works well on the other two platforms.

Could you please look into this and provide any insights or potential solutions?

**Comment by julesmarche** [comment]
Hi ! I have the same problem 👍
